addons/hr_recruitment_skills/views/hr_applicant_views.xml

```diff
@@ -51,4 +51,48 @@
             </filter>
         </field>
     </record>
+
+    <record id="crm_case_tree_view_inherit_hr_recruitment_skills" model="ir.ui.view">
+        <field name="name">hr.applicant.view.tree.inherit.skills</field>
+        <field name="model">hr.applicant</field>
+        <field name="inherit_id" ref="hr_recruitment.crm_case_tree_view_job"/>
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <xpath expr="//tree" position="attributes">
+                <attribute name="class" add="o_search_matching_applicant" separator=" "/>
```

Comment:
This is makes the option also present on the "all applicants" view, while it shouldn't IMO :thinking:
Additionally, why limit that button to kanban views when on an applicant view? It should apply to any view if the context makes sense for the button

Reply:
No, it won't show in the "All Applicants" view because it's flagged as primary and used in the Python action
The list, form, and kanban views all have the Cog menu in the Header that's why only kanban view has that button but
if you suggest then we can add that button in all views